force-app/main/default/lwc/geTemplateBuilderFormSection/geTemplateBuilderFormSection.js

```diff
@@ -16,6 +18,29 @@ export default class GeTemplateBuilderFormSection extends LightningElement {
     @api formSection;
     @api sourceObjectFieldsDescribe;
 
+    @api
```

Comment:
These also.  Is it that App needs to call the methods on section and section needs to call them on fields?  Just curious - did you look at or consider what a pub sub solution would look like?

Reply:
Yeah, absolutely on point there. I opted to go down the public method route as opposed to pubsub for three reasons.

1. We already have to query the DOM regardless of the communication method in order to set focus.
2. I didn't want to expand usage of our custom pubsub solution unless there was no other way to reduce to amount of code we'll need to retire when we move onto LMS. 
3. We're not locked into these public methods since neither of the involved components are exposed.

Reply:
OK, thanks for the details! Very helpful.